### ISSUE TITLE
The MultiSelect's clear button does not have an aria-label attribute

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#8025]
### Reproduction of the problem
1. Run this dojo example: https://dojo.telerik.com/VBprsrAf
2. Inspect the clear button (the element with class "k-clear-value"). 

### Current behavior
The button does not have an aria-label attribute.

### Expected/desired behavior
The button has an aria-label attribute.

**Workaround:**

```
<script>
    $(document).ready(function() {
        $(".k-clear-value").attr("aria-label", "Clear");
    })
</script>
```

### Environment

* **Kendo UI version:** 2024.3.1015
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by IvanDanchev** [comment]
Also applicable to the clear button in the AutoComplete, ComboBox, MultiColumnComboBox.

**Comment by alestoya** [comment]
As of the current situation, the [official MultiSelect ARIA specification](https://unpkg.com/browse/@progress/wct-a11y-spec@2.10.3/dist/jquery/aria/multiselect_aria.md) does not imply that the `aria-label` attribute should be displayed with the clear button. Furthermore, the button should not be part of the tab-sequence, which defeats the purpose of having the attribute there in the first place. 

The peculiarity applies to the AutoComplete, ComboBox, MultiColumnComboBox as well.
